content/docs/operations/sample-app.md

```diff
@@ -6,6 +6,18 @@ description: Deploy httpbin as a sample app.
 
 The httpbin app lets you test your API gateway by sending requests to it and receiving responses.
 
+Review the following diagram to understand the setup.
+
+```mermaid
+flowchart LR
+    A[client] -->|example.com| B[kgateway proxy]
+    B --> C[httpbin backend]
+```
+
+* The client calls the `www.example.com` hostname that you set up in the Gateway configuration.
+* The {{< reuse "docs/snippets/product-name.md" >}} proxy receives the request. Based on the routing rules that you set up in the Gateway configuration, the {{< reuse "docs/snippets/product-name.md" >}} proxy forwards the traffic to the backend destination, which is the httpbin service. The {{< reuse "docs/snippets/product-name.md" >}} proxy is available from an external LoadBalancer service that is backed by an IP address that your cloud provider typically assigns. For testing in a local cluster where you do not have an external service, you can enable port-forwarding so that the kgateway proxy listens on the localhost instead.
```

Comment:
```suggestion
* The {{< reuse "docs/snippets/product-name.md" >}} proxy receives the request. Based on the routing rules that you set up in the Gateway configuration, the {{< reuse "docs/snippets/product-name.md" >}} proxy forwards the traffic to the backend destination, which is the httpbin service. The {{< reuse "docs/snippets/product-name.md" >}} proxy is available from an external LoadBalancer service that is backed by an IP address that your cloud provider typically assigns. For testing in a local cluster where you do not have an external service, you can enable port-forwarding so that the {{< reuse "docs/snippets/product-name.md" >}} proxy listens on the localhost instead.
```